cmd/snap/cmd_run.go

```diff
@@ -461,7 +461,7 @@ func straceCmd() ([]string, error) {
 		stracePath,
 		"-u", current.Username,
 		"-f",
-		"-e", "!select,pselect6,_newselect,clock_gettime,sigaltstack,gettid",
+		"-e", "!select,pselect6,_newselect,clock_gettime,sigaltstack,gettid,gettimeofday",
```

Comment:
Can you leave a comment that gettimeofday makes arm64 strace hang?

Reply:
More VSDO issues?

Reply:
The comment sounds good, but I'm not so sure about the per-arch blacklist. That might lead to misleading ideas of differences across architectures which are not actually there. So unless the syscall is typically relevant for debugging purposes, having a single list sounds better.